Database.cs

```diff
@@ -128,7 +128,7 @@ public async Task BuildAsync(Func<string, string> modifyScript)
 
 			using (var conn = new SqlConnection(builder.ToString()))
 			{
-				await conn.ExecuteAsync($"if db_id('{database}') is null create database {database}");
+				await conn.ExecuteAsync($"if db_id('{database}') is null create database [{database}]");
```

Comment:
do you need it in the `if` statement, or does that not work?

Reply:
The `if` statement already has the database name delimited as a string. Bracket quoting is just T-SQL's way of allowing the use of non-alphanumeric characters in identifiers.